sys/crypto/psa_riot_cipher/Kconfig

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_PSA_RIOT_CIPHER_AES_128_ECB
+    bool
```

Comment:
Is it intended that all these modules are not selectable by the user?

Reply:
Yes, the user chooses the RIOT backend in `sys/psa_crypto/Kconfig` (as seen on #18578) and the rest is selected automatically.

Reply:
Ok then!